src/vmm/src/logger/logging.rs

```diff
@@ -754,16 +754,12 @@ mod tests {
         assert!(r.is_ok());
     }
 
-    #[test]
-    fn test_static_logger() {
-        log::set_max_level(log::LevelFilter::Info);
-        LOGGER.set_instance_id(TEST_INSTANCE_ID.to_string());
-
-        let mut reader = LOGGER.mock_init();
-
-        info!("info");
-        validate_log(&mut Box::new(&mut reader), "info\n");
-    }
+    // `test_static_logger` test has a conflict with the
+    // `vmm_config::test::test_itest_init_logger`
+    // which also uses tries to initialize logger and fail with an
+    // already initialized error.
+    // Since vmm_config::test_init_logger() cover the logger init test
+    // with more parameters, it is safe to remove test_static_logger().
```

Comment:
This comment should be in the commit body.
```suggestion
```

Reply:
I kept the comment for future reference so that no one tries to implement an init test.

Reply:
I think in this case the comment should be re-worded so as to not refer to removed code.

Reply:
Update the comment.